docs/source/models.rst

```diff
@@ -87,6 +90,62 @@ signature
 input_example
   reference to an artifact with :ref:`input example <input-example>`.
 
+databricks_runtime
+    Databricks runtime version and type, if the model was saved in a Databricks notebook or job.
+
+
+
+
+Additional Logged Files
+^^^^^^^^^^^^^^^^^^^^^^^
+For environment recreation, two additional files are automatically logged whenever a model is saved. These files can then be used to reinstall dependencies through either ``conda`` or ``pip``.
+
+conda.yaml
+    When saving a model, MLflow provides the option to pass in a conda environment parameter that can contain dependencies used by the model. If no conda environment is provided, a default environment is created based on the flavor of the model. This conda environment is then saved in ``conda.yaml``.
+requirements.txt
+    The requirements file is created from the pip dependencies inside of the conda environment. Additional pip dependencies can be added to ``requirements.txt`` by including them as a pip dependency in a conda environment and logging the model with the environment. 
```

Comment:
The first sentence is a little unclear imo - maybe we could rephrase like "the pip portion of the conda.yaml environment specification" and link to the conda docs on how `conda.yaml` is constructed